sensor_msgs/Image rgbImage
sensor_msgs/Image depthImage
int64 x1
int64 x2
int64 y1
int64 y2
